exercises/practice/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -12,32 +12,17 @@ import (
n interests; and that if he were enlightened, if his eyes were opened to his real normal interests, man would at once cease to do nasty things, would at once become good and noble because, being enlightened and understanding his real advantage, he would see his own advantage in the good and nothing else, and we all know that not one man can, consciously, act against his own interests, consequently, so to say, through necessity, he would begin doing good? Oh, the babe! Oh, the pure, innocent child! Why, in the first place, when in all these thousands of years has there been a time when man has acted only from his own interest? What is to be done with the millions of facts that bear witness that men, consciously, that is fully understanding their real interests, have left them in the background and have rushed headlong on another path, to meet peril and danger, compelled to this course by nobody and by nothing, but, as it were, simply disliking the beaten track, and have obstinately, wilfully, struck out another difficult, absurd way, seeking it almost in the darkness. So, I suppose, this obstinacy and perversity were pleasanter to them than any advantage.... Advantage! What is advantage? And will you take it upon yourself to define with perfect accuracy in what the advantage of man consists? And what if it so happens that a man's advantage, sometimes, not only may, but even must, consist in his desiring in certain cases what is harmful to himself and not advantageous. And if so, if there can be such a case, the whole principle falls into dust. What do you think–are there such cases? You laugh; laugh away, gentlemen, but only answer me: have man's advantages been reckoned up with perfect certainty? Are there not some which not only have not been included but cannot possibly be included under any classification? You see, you gentlemen have, to the best of my knowledge, taken your whole register of human advantages from the averages of statistical figures and politico-economical formulas. Your advantages are prosperity, wealth, freedom, peace–and so on, and so on. So that the man who should, for instance, go openly and knowingly in opposition to all that list would to your thinking, and indeed mine, too, of course, be an obscurantist or an absolute madman: would not he? But, you know, this is what is surprising: why does it so happen that all these statisticians, sages and lovers of humanity, when they reckon up human advantages invariably leave out one? They don't even take it into their reckoning in the form in which it should be taken, and the whole reckoning depends upon that. It would be no greater matter, they would simply have to take it, this advantage, and add it to the list. But the trouble is, that this strange advantage does not fall under any classification and is not in place in any list. I have a friend for instance ... Ech! gentlemen, but of course he is your friend, too; and indeed there is no one, no one to whom he is not a friend!`
```

Comment:
Remove a newline after the ` just so it's consistent with the others. In addition, I'm also wondering if we should split the text into several lines or alternatively have each text take just 1 line. What do you think @junedev ?

```suggestion
	dostoevsky3 = `But these are all golden dreams. Oh, tell me, who was it first announced, who was it first proclaimed, that man only does nasty things because he does not know his own interests; and that if he were enlightened, if his eyes were opened to his real normal interests, man would at once cease to do nasty things, would at once become good and noble because, being enlightened and understanding his real advantage, he would see his own advantage in the good and nothing else, and we all know that not one man can, consciously, act against his own interests, consequently, so to say, through necessity, he would begin doing good? Oh, the babe! Oh, the pure, innocent child! Why, in the first place, when in all these thousands of years has there been a time when man has acted only from his own interest? What is to be done with the millions of facts that bear witness that men, consciously, that is fully understanding their real interests, have left them in the background and have rushed headlong on another path, to meet peril and danger, compelled to this course by nobody and by nothing, but, as it were, simply disliking the beaten track, and have obstinately, wilfully, struck out another difficult, absurd way, seeking it almost in the darkness. So, I suppose, this obstinacy and perversity were pleasanter to them than any advantage.... Advantage! What is advantage? And will you take it upon yourself to define with perfect accuracy in what the advantage of man consists? And what if it so happens that a man's advantage, sometimes, not only may, but even must, consist in his desiring in certain cases what is harmful to himself and not advantageous. And if so, if there can be such a case, the whole principle falls into dust. What do you think–are there such cases? You laugh; laugh away, gentlemen, but only answer me: have man's advantages been reckoned up with perfect certainty? Are there not some which not only have not been included but cannot possibly be included under any classification? You see, you gentlemen have, to the best of my knowledge, taken your whole register of human advantages from the averages of statistical figures and politico-economical formulas. Your advantages are prosperity, wealth, freedom, peace–and so on, and so on. So that the man who should, for instance, go openly and knowingly in opposition to all that list would to your thinking, and indeed mine, too, of course, be an obscurantist or an absolute madman: would not he? But, you know, this is what is surprising: why does it so happen that all these statisticians, sages and lovers of humanity, when they reckon up human advantages invariably leave out one? They don't even take it into their reckoning in the form in which it should be taken, and the whole reckoning depends upon that. It would be no greater matter, they would simply have to take it, this advantage, and add it to the list. But the trouble is, that this strange advantage does not fall under any classification and is not in place in any list. I have a friend for instance ... Ech! gentlemen, but of course he is your friend, too; and indeed there is no one, no one to whom he is not a friend!`
```